templates/hypothesis.nomad

```diff
@@ -147,6 +147,37 @@ job "hypothesis" {
           destination = "local/usersync.py"
       }
 
+      template {
+        data = <<-EOF
+          #!/usr/bin/env python3
```

Comment:
Please move this into a separate file and include it here.

Take care with whitespace; check [here](https://github.com/liquidinvestigations/node/blob/60cee552d59745bc6b93d53c67253c08b38db35f/templates/rocketchat-migrate.nomad#L28-L29) for an example of how to include.

Throw all the scripts in a `scripts` subfolder, we'll refactor all the others in the future to separate scripts from nomad files and keep them in that subfolder.